ibm/ibm-how-to/launch-ubuntu-instances.rst

```diff
@@ -0,0 +1,153 @@
+Launch a Virtual Server Instance on IBM Cloud
+=============================================
+
+This tutorial guides you through the process of launching a Virtual Server Instance (VSI) on IBM Cloud using Ubuntu. Ensure you have an active IBM Cloud account before starting.
+
+Via Web Console
+---------------
+
+#. First, on the IBM Cloud Web Console, navigate to the Virtual Server creation form. To do this, navigate to     
+   :guilabel:`VPC Infrastructure` > :guilabel:`Compute`> :guilabel:`Virtual server instances`>, 
+   and then click on the blue :guilabel:`Create +`.  
+#. Enter a unique name for your VSI in the `Name` field.
+#. Under the image section, press on :guilabel:`Change image` and under the :guilabel:`Stock Images` tab, search for
+   "Ubuntu" in the search bar and select the Ubuntu OS you'd like to use. For the most recent OS, select :code:`ibm-ubuntu-24-04-minimal-amd64-1`
+#. Choose the desired profile for your VSI under the `Profile` section.
+#. Select an existing ssh key or create a new one by pressing on :guilabel:`Add SSH key` and following the instructions.
+#. Customize any other settings as needed.
+#. Press on the blue :guilabel:`Create virtual server` button to launch your VSI.
+
+Via IBM Cloud CLI
+-----------------
+
+**Prerequisites**
+
+- IBM Cloud account
+- IBM Cloud CLI installed on Ubuntu
+
+
+**Step 1: Log in to IBM Cloud**
+
+1. After installing the CLI, log in to your IBM Cloud account using the command:
+
+.. code-block:: bash
+
+   ibmcloud login
+
+2. If you have a federated ID, use:
+
+.. code-block:: bash
+
+   ibmcloud login --sso
+
+**Step 2: Target a Resource Group**
+
+1. List available resource groups:
+
+.. code-block:: bash
+
+   ibmcloud resource groups
+
+2. Target the desired resource group:
+
+.. code-block:: bash
+
+   ibmcloud target -g <Resource_Group_Name>
+
+**Step 3: Create SSH Key**
+
+1. Generate an SSH key for VSI access if you don't have one:
+
+.. code-block:: bash
+
+   ssh-keygen -t rsa -b 2048
+
+2. Add the SSH key to IBM Cloud:
+
+.. code-block:: bash
+
+   ibmcloud is key-create MyKey --public-key ~/.ssh/id_rsa.pub
+
+Replace "MyKey" with a unique name for your key.
+
+**Step 4: Gather Information for Your VSI**
+
+Before launching your Virtual Server Instance (VSI), you need to collect the VPC ID, Zone, Profile Name, and Image ID. Here's how to retrieve these details:
+
+1. **List VPCs to Get VPC ID:**
+
+First, you need the ID of the Virtual Private Cloud (VPC) where your instance will reside.
+
+.. code-block:: bash
+
+   ibmcloud is vpcs
+
+This command lists all VPCs in your account. Identify the VPC you wish to use and note its ID.
+
+2. **Identify the Availability Zone:**
+
+Your VSI needs to be hosted in an availability zone. To list available zones in your region:
+
+.. code-block:: bash
+
+   ibmcloud is regions
+
+Choose a region, then list the zones in that region:
+
+.. code-block:: bash
+
+   ibmcloud is zones <Region-Name>
```

Comment:
Should this say <region-id> instead since the twice prior step mentions to note the id, not the name?